src/components/ListComponent/ListComponent.css

```diff
@@ -0,0 +1,42 @@
+div {
+  width: 200px;
+}
+
+h2 {
+  font: 400 40px/1.5 Helvetica, Verdana, sans-serif;
```

Comment:
In case you've copied this style from somewhere read this https://www.webcredible.com/blog/css-shorthand-properties/